src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -8,7 +8,7 @@ const formConfig = {
   intentToFileUrl: '/evss_claims/intent_to_file/compensation',
   submitUrl: `${environment.API_URL}/v0/disability_compensation_form/submit`,
   trackingPrefix: 'disability-526EZ-',
-  formId: '21-526EZ',
+  formId: '21-526EZ-all-claims',
```

Comment:
We should think about this; changing it later is...gonna be a pain, I think.

Reply:
Yeah, I think I'd like to leave it as-is for now, but change it separately as soon as we can agree on a good id.